Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/riivolution/ui/RiivolutionViewHolder.java

```diff
@@ -32,18 +34,35 @@ public RiivolutionViewHolder(@NonNull View itemView)
     super(itemView);
 
     mTextView = itemView.findViewById(R.id.text_name);
-    mSpinner = itemView.findViewById(R.id.spinner_choice);
+    mChoiceLayout = itemView.findViewById(R.id.layout_choice);
+    mChoiceDropdown = itemView.findViewById(R.id.dropdown_choice);
+
+    if (mChoiceDropdown != null)
+    {
+      mChoiceDropdown.setSaveEnabled(false);
```

Comment:
Should this have the same comment about being a workaround for a bug as the equivalent code in ConvertFragment?